packages/block-library/src/navigation-link/link-ui/page-creator.js

```diff
@@ -85,10 +88,23 @@ export function LinkUIPageCreator( {
 					kind: 'post-type',
 				};
 
+				// Show success notice
+				createSuccessNotice( __( 'Page created successfully.' ), {
+					type: 'snackbar',
+					id: 'page-created-success',
+				} );
```

Comment:
What do you think about using the page name in the snackbar?

```suggestion
				createSuccessNotice(
					sprintf(
						__( '%s page created successfully.' ),
						decodeEntities( savedRecord.title.rendered )
					),
					{
						type: 'snackbar',
						id: 'page-created-success',
					}
				);

```

Reply:
Yes please. We should do this accounting for any edge case scenarios and i18n.

Reply:
Also should it be "successfully created" @scruffian?